server/tests/testAuthentication.js

```diff
@@ -25,17 +25,25 @@ describe("GET /api/currentuser", function() {
   //Status code 200 should have been sent.
 
   it("should return a 200 response if the user can log in", function(done) {
-    authenticatedUser.get("/api/login", function(err, response, body) {
+    authenticatedUser.get("http://localhost:8000/api/login", function(
```

Comment:
This is more clear than before. Would you be able to comment how the authenticate would work, that way anybody implementing this would understand? Just some food for thought